src/SelectableBox/tests/SelectableBoxSet.test.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
nit: is this eslint disable line still relevant?

Reply:
Not after rebasing on master, thanks 🙂